### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by findersen** [freekmurze/laravel-analytics#413] [Q&A]
Problem 1
    - google/apiclient[dev-master, v2.6.0, ..., v2.10.1] require guzzlehttp/psr7 ^1.2 -> found guzzlehttp/psr7[1.2.0, ..., 1.x-dev] but the package is fixed to 2.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - google/apiclient[v2.0.0-RC1, ..., v2.2.4] require monolog/monolog ^1.17 -> found monolog/monolog[1.17.0, ..., 1.x-dev] but the package is fixed to 2.3.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - google/apiclient[v2.3.0, ..., v2.5.0] require guzzlehttp/guzzle ~5.3.1||~6.0 -> found guzzlehttp/guzzle[5.3.1, ..., 5.3.x-dev, 6.0.0, ..., 6.5.x-dev] but it conflicts with your root composer.json require (^7.0.1).
    - spatie/laravel-analytics 3.11.0 requires google/apiclient ^2.0|^2.7 -> satisfiable by google/apiclient[v2.0.0-RC1, ..., 2.x-dev (alias of dev-master)].
    - google/apiclient 2.x-dev is an alias of google/apiclient dev-master and thus requires it to be installed too.
    - Root composer.json requires spatie/laravel-analytics ^3.11 -> satisfiable by spatie/laravel-analytics[3.11.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

---

Please tell me how I can solve the problem?

**Comment by Akhmami** [comment]
same
